### DISCUSSION TITLE
Microdot and Domain Example

### DISCUSSION THREAD

**Discussion by MalgudiLab** [miguelgrinberg/microdot#117] [Q&A]
Hello Team,

Is there way to attach domain name to microdot since IP address changes after connecting to wifi.

Thanks in advance,

**Comment by miguelgrinberg** [comment]
That is not done from the microcontroller's side.

One option that you have is to configure your Wi-Fi router to always give the same IP address to your device. Your router may also allow you to give the device a name, which any other devices in your LAN can then use to refer to it instead of the IP address. Basically you have to find out what your router can do in terms of addressing devices connected to Wi-Fi.
